test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs

```diff
@@ -752,25 +770,7 @@ protected static string GetDiagArg(string rootDir)
     /// Counts the number of logs following the '*.host.*' pattern in the given folder.
     /// </summary>
     protected static int CountTestHostLogs(string diagLogsDir, IEnumerable<string> testHostProcessNames)
-        => Directory.GetFiles(diagLogsDir, "*.host.*")
-            .Count(filePath =>
```

Comment:
This was looking into the test log and discriminating some of the log, because there were "more" testhosts than expected. But that was wrong. In some tests there are different counts between .net and .net framework target. This is because some hosts are Shared and some are not Shared.